_posts/2021-11-04-The-Problem-with-Economics.md

```diff
@@ -11,18 +11,17 @@ modified: 2020-11-04T16:03:49-04:00
+Economics is arguably a complex system more difficult to model than the weather. After all, water and carbon dioxide molecules behave in a predictable manner, i.e. a water molecule should do the same thing when exposed to the same circumstances repeatedly. People on the other hand do not behave like this, culture and upbring complicates the situation, along with the ability to remember past outcomes and therefore to constantly update their behaviour. 
 
 Measurements are much harder too. It’s reasonably straight forward to use weather balloons laden with scientific instruments to record the moisture content and temperature at different heights of the atmosphere in different locations. But macroeconomic data is no where near as detailed or reliable and this is something the discipline will likely just have to work around.
 
-Recognising the complexity of the challenge presented by Economics allows us to see that the approach taken to understanding it needs to replicate that taken by meteorology, with likely even more innovation and collaboration required to begin to build and create theories that are predictive and useful in the real world. 
+Recognising the complexity of the challenge presented by economics allows us to see that the approach taken to understanding it needs to replicate that taken by meteorology, with likely even more innovation and collaboration required to begin to build and create theories that are predictive and useful in the real world. 
 But this has not been the approach taken by macroeconomists. Macroeconomic research generally focuses on one specific area of the economy, attempting to first find a clear correlation between historical variables and then to use this correlation to make some future prediction. But as I have just outlined, an economy is a highly complex system that needs to be modelled as a whole if we are to understand the effects of any one area. 
 
-So why has economic research focused so much on small specific segments of the economy rather than examining and testing the whole system? I think this is likely a symptom of how research is structured, with economists all working individually or at best in small teams. Building a global economic model is far too large a task for even a small team to achieve, and hence there is no choice for economic researchers other than to adopt an approach that is destined to fail from the outset. 
-
+So why has economic research focused so much on small specific segments of the economy rather than examining and testing the whole system? I think this is likely a symptom of how research is structured, with economists all working individually or at best in small teams. Building a global economic model is far too large a task for even a small team to achieve, and hence there is no choice for economic researchers other than to adopt an approach that is destined to fail from the outset.
```

Comment:
Reads as an argument that economics are more complex than weather, economics cannot be measured with expected outcomes the way the weather can, but then says to apply the same approach as weather prediction to economics prediction.
When you come to the piece saying that macroeconomic research generally focuses on a specific area of the economy and looks at correlation between historical variables, that to me seems like a reasonable approach given what you've stated above 
I think to make your point, emphasize more about how shifts in the system elsewhere greatly affect other parts and give a clear economic example because your example right now is generic and weather focused. You could also mention how historical events such as political wars can cause shifts in economic patterns and maybe that's why looking at historical correlations isn't the most sound (but then part of your project also does that so maybe don't shoot yourself in the foot)